include/irq.h

```diff
@@ -163,16 +163,17 @@ irq_connect_dynamic(unsigned int irq, unsigned int priority,
  *
  * Example usage:
  *
- * ISR_DIRECT_DECLARE(my_isr)
- * {
- *	bool done = do_stuff();
- *	ISR_DIRECT_PM(); <-- done after do_stuff() due to latency concerns
- *	if (!done) {
- *		return 0;  <-- Don't bother checking if we have to z_swap()
- *	}
- *	k_sem_give(some_sem);
- *	return 1;
- * }
+ *     ISR_DIRECT_DECLARE(my_isr)
+ *     {
+ *             bool done = do_stuff();
+ *             ISR_DIRECT_PM(); // done after do_stuff() due to latency concerns
```

Comment:
`//` style comments are not used in Zephyr

Reply:
Yes, you are right and I thought about it before.
But that docs are generated from the .h file, in that file
structure looks like

```
/**
 * Example usage:
 *
 *     ISR_DIRECT_DECLARE(my_isr)
 *     {
 *             bool done = do_stuff();
 *             // done after do_stuff() due to latency concerns
 *             ISR_DIRECT_PM();
 *             if (!done) {
 *                 return 0; // don't bother checking if we have to z_swap()
 *             }
 *
 *             k_sem_give(some_sem);
 *             return 1;
 *      }
 *
 * @param name symbol name of the ISR
 */
```
If I will try to add /* */ comments there 
```
/**
 * Example usage:
 *
 *     ISR_DIRECT_DECLARE(my_isr)
 *     {
 *             bool done = do_stuff();
 *             /* done after do_stuff() due to latency concerns */
 *             ISR_DIRECT_PM();
 *             if (!done) {
 *                 /* don't bother checking if we have to z_swap() */
 *                 return 0; 
 *             }
 *
 *             k_sem_give(some_sem);
 *             return 1;
 *      }
 *
 * @param name symbol name of the ISR
 */
```
the last closing comment tag */ below that last line  `* @param name symbol name of the ISR` won't be detected, as a result construction won't work. 
I saw that other examples in the documentation pages which are generated from the .h file are using `//` comments. If you know an answer how to solve it, it will be great.